components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -258,7 +259,14 @@ export default function (props: { project?: Project; isAdminDashboard?: boolean
                                                         alt={p.info.startedBy}
                                                     />
                                                 )}
-                                                Triggered {formatDate(p.info.startedAt)}
+                                                <Tooltip
+                                                    className="w-fit"
+                                                    content={`Triggered At ${dayjs(p.info.startedAt)
```

Comment:
**suggestion**: I'd suggest removing the prefixes from all tooltip instances[[1](https://github.com/gitpod-io/gitpod/pull/15812/files#diff-5374e0e0ce47926ea366af7e62b2a2b1a9c598f1a02c0128631a2fe52fa77251R264)][[2](https://github.com/gitpod-io/gitpod/pull/15812/files#diff-5374e0e0ce47926ea366af7e62b2a2b1a9c598f1a02c0128631a2fe52fa77251R294)][[3](https://github.com/gitpod-io/gitpod/pull/15812/files#diff-568d9692491d69894e48071932f4aae5ce22fb5e97df1f46853fec552783ebb2R361)][[4](https://github.com/gitpod-io/gitpod/pull/15812/files#diff-d6b12480648fcb6a8374fa6ea08410892c9477c73731358fff61154d6f647542R98)][[5](https://github.com/gitpod-io/gitpod/pull/15812/files#diff-53ff0bdfe65ca0d0e6a4af4ac305ab6a4f94c852f14e509b915659e8d653c820R240)][[6](https://github.com/gitpod-io/gitpod/pull/15812/files#diff-02ffe6851cca7925757e521dfba549b1e72892581cc6a6596cb4f5c0656cb300R193)].